packages/fxa-settings/src/pages/Signin/SigninTokenCode/index.tsx

```diff
@@ -208,7 +208,7 @@ const SigninTokenCode = ({
   const cmsInfo = integration?.getCmsInfo();
 
   return (
-    <AppLayout>
+    <AppLayout integration={integration}>
```

Comment:
In hindsight, we should be passing the `cmsInfo` object to the `AppLayout`. That change is a much bigger refactor.  Filed https://mozilla-hub.atlassian.net/browse/FXA-12172 to fix